resteasy-client/src/main/java/org/jboss/resteasy/plugins/providers/sse/client/SseEventSourceImpl.java

```diff
@@ -126,13 +135,21 @@ private SseEventSourceImpl(final WebTarget target, String name, long reconnectDe
       {
          name = String.format("sse-event-source(%s)", target.getUri());
```

Comment:
Name is never used, can you please check it? I know, that this line is not changed in this PR ...

Reply:
@marekkopecky, I'll ask Jim about this one. It looks like he might have intended to use it.